cmd/podgrouper/app/options.go

```diff
@@ -35,7 +35,7 @@ func (o *Options) AddFlags(fs *flag.FlagSet) {
 	fs.BoolVar(&o.SearchForLegacyPodGroups, "search-legacy-pg", true, "If this flag is enabled, try to find pod groups with legacy name format. If they exist, use the found pod groups instead of creating new once with current name format")
 	fs.BoolVar(&o.KnativeGangSchedule, "knative-gang-schedule", true, "Schedule knative revision as a gang. Defaults to true")
 	fs.StringVar(&o.SchedulerName, "scheduler-name", "kai-scheduler", "The name of the scheduler used to schedule pod groups")
-	fs.StringVar(&o.SchedulingQueueLabelKey, "queue-label-key", "kai.scheduler/queue", "Scheduling queue label key name")
```

Comment:
Dont you prefer to keep it kai and revert it in the engine + kai helm chart?

That way you have single place to change when the breaking changes day will come

Reply:
If we change it to kai now it will introduce a breaking change, the plan is to coordinate a few of them and change together soon.

Reply:
But you can configure it to run.ai in the helm chart default value

Reply:
We decided to keep it here for now